devito/finite_difference.py

```diff
@@ -137,20 +137,27 @@ def first_derivative(*args, **kwargs):
     dim = kwargs.get('dim', x)
     diff = kwargs.get('diff', h)
     order = kwargs.get('order', 1)
-    side = kwargs.get('side', 1)
+    side = kwargs.get('side', "centered")
```

Comment:
I don't believe @FabioLuporini  was asking you to move to string comparisons. Only to use constants with more descriptive names, rather than numbers like 0 and 1 which don't mean anything to an average programmer who is focussed on another part of the code.